### PR TITLE
Completed work on EHPR-281 Add confirmation page for profile update

### PR DESCRIPTION
**Pull request by dbayly-freshworks**
Updated copy on landing page
Added confirmation page for after submission update.

### CODE REVIEW

File: apps/web/src/pages/submission/[step].tsx [hunk 1]
```diff
@@ -67,8 +61,10 @@ const Submission = () => {
             </p>
 
             <p className='font-bold'>
-              If you have any questions about our collection or use of personal information, please
-              email your inquiries to&nbsp;
+              <b>
```
Comment (coryatwater-FW) [inline apps/web/src/pages/submission/[step].tsx:64]: could probably remove this `<b>` tag

File: apps/web/src/pages/update-submission/confirmation.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+const Confirmation: React.FC = () => {
+  const { push, query, isReady } = useRouter();
+  const id: string = query.id as string;
+
+  useEffect(() => {
+    // redirect to the home page if the id is not present
+    if (isReady && !id) {
+      push('/');
+    }
+  });
```
Comment (coryatwater-FW) [inline apps/web/src/pages/update-submission/confirmation.tsx:14]: how come the missing dependency as opposed to `id`?

Comment (dbayly-freshworks) [reply apps/web/src/pages/update-submission/confirmation.tsx:14]: I can clear this out. I copied this template from the confirmation page for the main form. An ID is not needed here.

Comment (coryatwater-FW) [reply apps/web/src/pages/update-submission/confirmation.tsx:14]: ah ok, was just curious if it was on purpose

**Review by coryatwater-FW — APPROVED** [review]

**Pull request merged**